docs/cloud-native-security/session-view.asciidoc

```diff
@@ -80,8 +81,7 @@ It also shows the Linux user "Ubuntu" started the session.
 6. The *Child processes* button. Click to expand or collapse a process’s children.
 You can also expand collapsed alerts and scripts where they appear.
 Collapsed processes will automatically expand when their contents match a search.
-7. The *Alerts* button. Click to show alerts caused by the parent process. Note the red line to the left
-of the process that caused the alert.
+7. The *Alerts* button. Click to show alerts caused by the parent process. The `(2)` indicates that there are two alerts. Note the red line to the left of the event that caused the alert. Both alerts caused by this event are `process` alerts, as indicated by the gear icon.
```

Comment:
Most of the descriptions are generic, so it would help to signal when we're talking about something specific in the screenshot.
```suggestion
7. The *Alerts* button. Click to show alerts caused by the parent process. In this example, the `(2)` indicates that there are two alerts. Note the red line to the left of the event that caused the alert. Both alerts caused by this event are `process` alerts, as indicated by the gear icon.
```